regions/io/crtf/tests/test_crtf.py

```diff
@@ -204,3 +204,11 @@ def test_angle_serialization():
     expected = ('#CRTFv0\nglobal coord=J2000\ncircle[[10.000009deg, '
                 '20.000002deg], 0.000278deg]\n')
     assert regstr == expected
+
+
+def test_read_sexagesimal_regression407():
+    filename = 'data/crtf_carta_sexagesimal.crtf'
+    filename = get_pkg_data_filename(filename)
+    regs = Regions.read(filename, errors='warn', format='crtf')
+
+    assert regs[0].center.ra == 281.93342096*u.deg
```

Comment:
```suggestion
    assert_quantity_allclose(regs[0].center.ra, 281.93342096*u.deg, rtol=1e-9)
```
Looked a bit suspicious with the float comparison, but that was a nasty one lurking just past the default printed digits! Will need
`from astropy.tests.helper import assert_quantity_allclose`
at the top (though `np.allclose` should work now with quantities for all supported numpy versions as well).